vendor/src/github.com/godbus/dbus/conn.go

```diff
@@ -200,7 +200,7 @@ func (conn *Conn) Close() error {
 
```

Comment:
Changing this file like this isn't allowed. You need to send a pull request against that `github.com/godbus/dbus` repository first and update the vendored version.